libjob_queue/src/lsf_driver.c

```diff
@@ -919,12 +923,16 @@ void * lsf_driver_submit_job(void * __driver ,
       lsf_submit_method_enum submit_method = driver->submit_method;
       pthread_mutex_lock( &driver->submit_lock );
 
-      if (driver->debug_output)
-        printf("LSF DRIVER submitting using method:%d \n",submit_method);
+      if (driver->debug_output){
+        printf("LSF DRIVER submitting using method:%d \n",submit_method); //TODO: remove?
+        res_log_add_fmt_message(LOG_INFO, NULL ,"LSF DRIVER submitting using method:%d \n",submit_method);
```

Comment:
I think the `res_log_add_fmt_message( ... )` call should be unconditional - i.e. outside the `if (driver->debug_output)`test.